src/Microsoft.AspNetCore.Mvc.Razor/Internal/DefaultRazorPageFactoryProvider.cs

```diff
@@ -62,10 +78,12 @@ public RazorPageFactoryResult CreateFactory(string relativePath)
             var result = CompilerCache.GetOrAdd(relativePath, _compileDelegate);
             if (result.Success)
             {
+                _logger?.PrecompiledViewFound();
```

Comment:
This isn't correct. Not all views are precompiled views. That said, it might be better to print this in as part of `RazorViewEngine`: https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngine.cs#L305. In the current usage, the other message would appear after this one which might be confusing.